src/BuiltInTools/dotnet-format/dotnet-format.csproj

```diff
@@ -33,7 +33,14 @@
     <PackageReference Include="Microsoft.CodeAnalysis" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" />
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic" />
+
     <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.MSBuild" />
+    <!-- Suppress transitive dependencies of MSBuildWorkspace on MSBuild to avoid unnecessary copies -->
+    <PackageReference Include="Microsoft.Build" ExcludeAssets="runtime" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Build.Framework" ExcludeAssets="runtime" PrivateAssets="All" />
```

Comment:
Definitely test if you haven't whether this particular one is suppressable. I could imagine it breaking if the assembly is missing if we need to load ILogger.